src/Util/ParseUrl.php

```diff
@@ -186,6 +210,12 @@ public static function getSiteinfo($url, $no_guessing = false, $do_oembed = true
 			return $siteinfo;
 		}
 
+		$type = self::getContentType($url);
```

Comment:
Ok, now we probably don't need lines 231-239 that were doing the same thing on the full request. This content type parsing was skipped for media files bigger than 1MB on line 227.

Reply:
When testing I had significant speed impacts when fetching information for videos. For videos the system needed about 2-4 seconds to fetch information while `head` returns it immediately.

Reply:
I agree, but now lines 231-239 are superfluous and I can't remove them myself from the GitHub interface through suggestions.

Reply:
done - with some more changes.